### PR TITLE
feat: apply dopplerSystemUsage in firstSteps section

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Apply dopplerSystemUsage service in first step** 
**Jira Ticket:** https://makingsense.atlassian.net/browse/DAT-1105
**Note**: this is a quick fix to the crossed in `step 4`. Ideally all data will come from the new [system summary api](https://github.com/FromDoppler/doppler-system-usage)

**Description**

When an user visited the `reports` page and has sent campaigns, the `step 4 `is crossed ..

### CODE REVIEW

File: src/components/Dashboard/FirstSteps/index.js [hunk 1]
```diff
@@ -14,63 +14,74 @@ import {
...
+    useEffect(() => {
+      // TODO: use getUserSystemUsage for the same purpose as getSystemUsageSummaryData
+      // TODO: remove systemUsageSummary service
+      const fetchData = async () => {
+        dispatch({ type: FIRST_STEPS_ACTIONS.START_FETCH });
+        const response = await Promise.all([
+          systemUsageSummary.getSystemUsageSummaryData(),
+          dopplerSystemUsageApiClient.getUserSystemUsage(),
+        ]);
+        const [systemUsageResponse, dopplerSystemUsageResponse] = response;
```
Comment (andresmoschini) [inline src/components/Dashboard/FirstSteps/index.js:37]: What do you think about 

```suggestion
        const responses = await Promise.all([
          systemUsageSummary.getSystemUsageSummaryData(),
          dopplerSystemUsageApiClient.getUserSystemUsage(),
        ]);
        const [systemUsageResponse, dopplerSystemUsageResponse] = responses;
```

Or

```suggestion
        const [systemUsageResponse, dopplerSystemUsageResponse] = await Promise.all([
          systemUsageSummary.getSystemUsageSummaryData(),
          dopplerSystemUsageApiClient.getUserSystemUsage(),
        ]);
```

Comment (jcamposmk) [reply src/components/Dashboard/FirstSteps/index.js:37]: It's better! 👍

File: src/components/Dashboard/FirstSteps/index.js [hunk 2]
```diff
@@ -14,63 +14,74 @@ import {
...
+        if (systemUsageResponse.success && dopplerSystemUsageResponse.success) {
+          const { value: data } = systemUsageResponse;
+          const dataMapped = mapSystemUsageSummary({
+            ...data,
+            ...dopplerSystemUsageResponse.value,
+          });
```
Comment (andresmoschini) [inline src/components/Dashboard/FirstSteps/index.js:43]: What do you think about 

```suggestion
          const dataMapped = mapSystemUsageSummary({
            ...systemUsageResponse.value,
            ...dopplerSystemUsageResponse.value,
          });
```

Or

```suggestion
          const { value: oldData } = systemUsageResponse;
          const { value: newData } = dopplerSystemUsageResponse;
          const dataMapped = mapSystemUsageSummary({
            ...oldData,
            ...newData,
          });
```

Comment (andresmoschini) [reply src/components/Dashboard/FirstSteps/index.js:42]: Another option is merging all of this in the `dopplerSystemUsageApiClient`

**Pull request merged**